jobs/temp-mini-e2e-helm.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- project:
+    name: temp-mini-e2e
```

Comment:
Looks good, but can you rename it to something suitable? Maybe `mini-e2e-helm-sc` or something, `temp` just does not say anything.

Also make sure to add a TODO in the original issue so that you are reminded to remove this job once it is part of the standard once(s).

Reply:
Thanks for the suggestion! Will do the needful

Reply:
Done https://github.com/ceph/ceph-csi/pull/1786#issue-539375046